pkg/controller/registry/resolver/steps.go

```diff
@@ -6,11 +6,13 @@ import (
 	"fmt"
 	"strings"
 
+	v1 "github.com/operator-framework/api/pkg/operators/v1"
```

Comment:
nit: import order, and extra `v1`.

I suggest just aliasing it to `operatorsv1`, and `v1alpha1` to `operatorsv1alpha1` if we feel it makes sense to do in this PR.

Reply:
No longer applies but taking the lesson and applying it to work in the future.